legion/legion/http.py

```diff
@@ -19,15 +19,41 @@
 import functools
 import os
 import logging
+import urllib
 
 import legion.config
 import legion.utils
 
 import flask
+from requests.compat import urlencode
+from requests.utils import to_key_val_list
+from urllib.parse import parse_qs
 
 LOGGER = logging.getLogger(__name__)
 
 
+def encode_http_params(data):
```

Comment:
Should not this be in the client module?

Reply:
Yeah, but model client is a part of legion package, so all functionalities related to HTTP protocol placed here.